commodore/cluster.py

```diff
@@ -13,14 +13,79 @@
 )
 
 from .config import Config
-
-
-def fetch_cluster(cfg, clusterid):
-    cluster = lieutenant_query(cfg.api_url, cfg.api_token, "clusters", clusterid)
-    # TODO: move Commodore global defaults repo name into Lieutenant
-    # API/cluster facts
-    cluster["base_config"] = "commodore-defaults"
-    return cluster
+from .git import HEAD
+
+
+class Cluster:
```

Comment:
Consider adding type hints for the class fields

```suggestion
class Cluster:
    __cfg: Config
    __cluster_response: Dict[str, Any]
    __tenant_response: Dict[str, Any]
```